apps/settings/locales/settings.en-US.properties

```diff
@@ -1274,6 +1274,7 @@ copypaste=Copy/Paste
 multi-screen=Multi-Screen
 verbose-app-permissions=Verbose App Permissions
 pinning-the-web=Enable Pinning the Web
+view-source=Enable View Source Gesture
```

Comment:
for the header title in the system app? or is the key already there? (it's empty on my phone)